examples/pytorch/question-answering/trainer_qa.py

```diff
@@ -56,6 +56,11 @@ def evaluate(self, eval_dataset=None, eval_examples=None, ignore_keys=None):
             eval_preds = self.post_process_function(eval_examples, eval_dataset, output.predictions)
             metrics = self.compute_metrics(eval_preds)
 
+            # Prefix all keys with metric_key_prefix + '_'
+            for key in list(metrics.keys()):
+                if not key.startswith(f"{metric_key_prefix}_"):
+                    metrics[f"{metric_key_prefix}_{key}"] = metrics.pop(key)
```

Comment:
This shouldn't be necessary here, instead pass along the `metric_key_prefix` to the `eval_loop` above.

Reply:
Same deal here -- metrics are disabled before `eval_loop`